packages/block-library/package.json

```diff
@@ -65,6 +66,7 @@
 		"@wordpress/wordcount": "file:../wordcount",
 		"change-case": "^4.1.2",
 		"classnames": "^2.3.1",
+		"codemirror": "6.0.1",
```

Comment:
I wonder if some linting to prevent this being imported the traditional way would be beneficial to ensure codemirror isn't accidentally bundled in the future.

I think there are some restricted import statements already for particular packages, but I'm not familiar with how that's implemented. 🤔

Reply:
Thanks for your suggestion :) I addressed it in https://github.com/WordPress/gutenberg/pull/60155/commits/421f2d4ba1256ad97198a2b8004d14af5915e970